client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.jsx

```diff
@@ -0,0 +1,273 @@
+import { Gridicon } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import Badge from 'calypso/components/badge';
+import Tooltip from 'calypso/components/tooltip';
+
+const getLinks = ( type, status, siteUrl, siteId ) => {
```

Comment:
Can we include unit tests for this function in a follow-up PR?

Reply:
Sure